src/DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -2723,31 +2730,31 @@ static simulated function string GetBingoGoalHelpText(string event,int mission,
         case "JocksToilet":
             return "Use the toilet in Jock's Tonnochi Road apartment.  The bathroom is behind a sliding door next to the kitchen.";
         case "Greasel_ClassDead":
-            return "Kill enough greasels.  You must kill the greasels yourself.";
+            return "Kill enough greasels.";
         case "support1":
             return "Blow up the gas station.";
         case "UNATCOTroop_ClassDead":
-            return "Kill enough UNATCO Troopers.  You must kill them yourself.";
+            return "Kill enough UNATCO Troopers.";
         case "Terrorist_ClassDead":
-            return "Kill enough NSF Troops.  You must kill them yourself.";
+            return "Kill enough NSF Troops.";
         case "MJ12Troop_ClassDead":
-            return "Kill enough MJ12 Troopers.  You must kill them yourself.";
+            return "Kill enough MJ12 Troopers.";
         case "MJ12Commando_ClassDead":
-            return "Kill enough MJ12 Commandos.  You must kill them yourself.";
+            return "Kill enough MJ12 Commandos.";
         case "Karkian_ClassDead":
-            return "Kill enough karkians.  You must kill them yourself.";
+            return "Kill enough karkians.";
         case "MilitaryBot_ClassDead":
-            return "Destroy enough military bots.  You must destroy them yourself.  Disabling them with EMP grenades does not count.";
+            return "Destroy enough military bots.  Disabling them with EMP does not count.";
```

Comment:
The change to be more generally "EMP" is good (but same comment about the removal of the destroying them yourself still applies)